nix/shared.nix

```diff
@@ -218,7 +212,7 @@ let
     );
   };
 
-  overlayGHC802 = pkgsNew: pkgsOld: {
+  overlayGHC822 = pkgsNew: pkgsOld: {
```

Comment:
I haven't actually checked whether the overlay is still necessary in this form. It doesn't seem to hurt though…